doc/en/how-to/usage.rst

```diff
@@ -168,15 +168,17 @@ You can invoke ``pytest`` from Python code directly:
 
 .. code-block:: python
 
-    pytest.main()
```

Comment:
In this snippet I think it makes more sense to have this instead:

```python
retcode = pytest.main()
```

Here we shouldn't assume that this will be called as the "main" part of a script (might be confusing).